### ISSUE TITLE
Investigate binders health indicator property

### ISSUE THREAD

**Issue by ilayaperumalg** [spring-cloud/spring-cloud-stream#950]
Currently, in the docs we have:

```
Spring Cloud Stream provides a health indicator for binders. It is registered under the name of binders and can be enabled or disabled by setting the management.health.binders.enabled property.

```

By default the binders health indicator is enabled and setting it to `false` disables the binders health indicator configuration

But, setting `management.health.binders.enabled=true` doesn't seem to work.

**Comment by viniciusccarvalho** [comment]
Hi @ilayaperumalg I tried this with `1.3.0-BUILD-SNAPSHOT` and I was able to get it working as expected.

If no `management.health.binders.enabled` property is present, the health bean is present (I'm assuming is matching on missing)

If `management.health.binders.enabled=false` then, no health indicator is shown on `/health`
Setting `management.health.binders.enabled=true` enables the health indicator back.

Am I missing something here?
